dashboard/src/components/Config/AppRepoList/AppRepoForm.tsx

```diff
@@ -372,6 +395,72 @@ export function AppRepoForm(props: IAppRepoFormProps) {
           </Row>
         </div>
       </div>
+      <div className="clr-form-control">
+        <label className="clr-control-label">Repository Type</label>
+        <span className="clr-form-description">
+          Charts can be stored either in a Helm repository (e.g. ChartMuseum) or in a OCI Registry.
+          Select the one that applies.
```

Comment:
"Select which type of repository you are adding." (though we're confusing two different types of repositories here, another option:
"Select which type of chart catalog you are adding."

Reply:
"Select the chart storage type"?

Reply:
I am going to merge this but let me know if you don't agree, I can open another PR for this.